PWG/muon/AliAnalysisMuMuFlowEP.h

```diff
@@ -0,0 +1,110 @@
+#ifndef ALIANALYSISMUMUFLOWEP_H
+#define ALIANALYSISMUMUFLOWEP_H
+
+/**
+ *
+ * \class AliAnalysisMuMuNch
```

Comment:
Clear give away of a cut-and-paste ? ;-)
Please fix at least this part of the (almost) missing documentation...